content/source/docs/extend/schemas/schema-methods.html.md

```diff
@@ -1,9 +1,9 @@
 ---
 layout: "extend"
-page_title: "Home - Extending Terraform"
+page_title: "Home - Building Terraform Providers"
```

Comment:
```suggestion
page_title: "Building Terraform Providers - Schemas Methods"
```

Reply:
Shouldn't it be "schema"?

Reply:
The h1 header in the Markdown was using the plural "Terraform Schemas Methods" -- I personally think its better singular too, but the content and the metadata should match either which way.

Reply:
I'm a fan of Schema but I realize no one asked me for my opinion, so feel free to disregard